### PR TITLE
Add new folks

### PR DESCRIPTION
**Pull request by muratg**

### CODE REVIEW

File: src/ProjectKIssueList/Models/StaticPersonSetProvider.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class StaticPersonSetProvider : IPersonSetProvider
                         "NTaylorMullen",
                         "pranavkm",
                         "rynowak",
+                        "ryanbrandenburg",
```
Comment (Eilon) [inline src/ProjectKIssueList/Models/StaticPersonSetProvider.cs:21]: Sort!

File: src/ProjectKIssueList/Models/StaticPersonSetProvider.cs [hunk 2]
```diff
@@ -40,6 +41,7 @@ public class StaticPersonSetProvider : IPersonSetProvider
                         "Tratcher",
                         "troydai",
                         "victorhurdugaci",
+                        "pakrym",
```
Comment (Eilon) [inline src/ProjectKIssueList/Models/StaticPersonSetProvider.cs]: Sort!

Comment (muratg) [inline src/ProjectKIssueList/Models/StaticPersonSetProvider.cs]: image

**Comment by Eilon** [comment]
Squash please :smile:

**Comment by muratg** [comment]
Just edited the file on the GitHub UI. :)  Too lazy to squash which would require me to resend a PR (because my initial fork was too old) so not worth the effort. Do it on your next commit. :)

**Comment by Eilon** [comment]
Hah, will do :smile:

**Pull request closed without merging**